src/erc20/optimistic.json

```diff
@@ -86,5 +86,235 @@
       "decimals": 18,
       "chainId": 10,
       "logoURI": "https://raw.githubusercontent.com/sushiswap/icons/master/token/slink.jpg"
-    }   
+    },
+    {
+      "chainId": 10,
+      "address": "0x68f180fcCe6836688e9084f035309E29Bf0A2095",
+      "name": "Wrapped Bitcoin",
+      "symbol": "WBTC",
+      "decimals": 8,
+      "logoURI": "https://ethereum-optimism.github.io/logos/WBTC.svg",
+      "extensions": {
```

Comment:
Remove this unknown field for every token.

Reply:
done